nilearn/plotting/html_surface.py

```diff
@@ -190,10 +190,11 @@ def colorscale(cmap, values, threshold=None, symmetric_cmap=True):
     cmap = mpl_cm.get_cmap(cmap)
     abs_values = np.abs(values)
     if symmetric_cmap:
-        vmax = abs_values.max()
-        vmin = - vmax
+        vmax = fast_abs_percentile(abs_values, 99.7)
+        vmin = -vmax
     else:
-        vmin, vmax = values.min(), values.max()
+        vmin = fast_abs_percentile(abs_values, 0.3)
```

Comment:
I think that this is the most controversial change: so far, color maps were symmetric about 0. Now, they no longer are. This may surprise users. Any opinion on that

Reply:
They are symmetric as long as `symmetric_cmap` is True, which is the default option. In the case that `symmetric_cmap` is False, then the original code also took the min and max values, leading to a non symmetric cmap --  this PR does not change the behaviour wrt the original code.